src/components/Messages/MessageList/styles.scss

```diff
@@ -33,3 +33,86 @@
 		list-style: none;
 	}
 }
+
+.call {
+	position: absolute;
+	top: 0;
+
+	display: flex;
+	flex-direction: column;
+
+	width: 100%;
+	height: 50%;
+	margin-bottom: 5px;
+
+	padding-top: 40px;
+	padding-bottom: 0;
+
+	text-align: center;
+
+	color: #ffffff;
+
+	border: 1px solid black;
+
+	background: #1f2329;
+
+	font-family: Inter;
```

Comment:
Lets not override the global font family. We should try to keep the font consistent across entire app